ImperatorToCK3/Source/Imperator/Countries/Country.h

```diff
@@ -55,9 +58,7 @@ namespace Imperator
 			void setCK3Title(const std::shared_ptr<CK3::Title>& theTitle) { ck3Title = theTitle; }
 
 		private:
-			void registerKeys();
-
-			unsigned long long countryID = 0;
+			unsigned long long ID = 0;
```

Comment:
I have a mild preference for uint64_t as it doesn't require looking up or memorizing the size of a long long. But it's not a big deal either way. Requires including cstdint